internal/client.go

```diff
@@ -835,6 +839,116 @@ type (
 		NonRetryableErrorTypes []string
 	}
 
+	// Priority contains metadata that controls relative ordering of task processing
```

Comment:
This is _almost_ verbatim the [(excellent) documentation from the protos](https://github.com/temporalio/api/pull/513/files#diff-de60adc512f72d87dc1e38c0f30b41a16dede9243dfaf6209b4af9b5447fa551R250). Only the first and last sentence were slightly edited.